tests/report/common/tests.cpp

```diff
@@ -122,19 +123,18 @@ oe_result_t oe_get_collaterals(
  * Verify the integrity of the report and its signature,
  * with optional collateral data that is associated with the report.
  *
- * This function verifies that the report signature is valid. If the report is
- * local, it verifies that it is correctly signed by the enclave
- * platform. If the report is remote, it verifies that the signing authority is
- * rooted to a trusted authority such as the enclave platform manufacturer.
+ * This function verifies that the report signature is valid. This only applies
+ * to remote reports.  For remote reports it verifies that the signing authority
+ * is rooted to a trusted authority such as the enclave platform manufacturer.
  *
- * @param enclave The instance of the enclave that will be used to
- * verify a local report. For remote reports, this parameter can be NULL.
- * @param report The buffer containing the report to verify.
- * @param report_size The size of the **report** buffer.
- * @param collaterals The collateral data that is associated with the report.
- * @param collaterals_size The size of the **collaterals** buffer.
- * @param parsed_report Optional **oe_report_t** structure to populate with the
- * report properties in a standard format.
+ * @param[in] enclave The instance of the enclave that will be used.
+ * @param[in] report The buffer containing the report to verify.
+ * @param[in] report_size The size of the **report** buffer.
+ * @param[in] collaterals Optional The collateral data that is associated with
+ * the report.
+ * @param[in] collaterals_size The size of the **collaterals** buffer.
+ * @param[out] parsed_report Optional **oe_report_t** structure to populate with
```

Comment:
there's no comment for the the `input_validation_time` parameter.

Reply:
Fixed.